.github/workflows/unit-integration-test.yml

```diff
@@ -0,0 +1,97 @@
+name: GitHub Actions Workflow
+
+on:
+  pull_request:
+  workflow_dispatch: # Allow manual trigger
+
+jobs:
+  build:
+    runs-on: self-hosted
+
+    env:
+      CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+      E2E_ENVIRONMENT: 'CI'
+      AB2D_V2_ENABLED: 'true'
+      SEARCH_BENE_BATCH_SIZE: '10'
+      AWS_DEFAULT_REGION: "us-east-1"
+      WORKSPACE: ${{ github.workspace }}
+      AB2D_BFD_KEYSTORE_LOCATION: "${{ github.workspace }}/opt/ab2d/ab2d_bfd_keystore"
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: temurin
+          cache: maven
+
+      - name: Download bfd keystore
+        run: |
+          aws s3 cp s3://bcda-opensbx-access-logs/ab2d_sbx_keystore /tmp/ab2d_sbx_keystore
```

Comment:
I'm confused by the `${{ github.workspace }}/opt/ab2d/ab2d_bfd_keystore` in the AB2D_BFD_KEYSTORE_LOCATION above, then this downloads to /tmp/ab2d_sbx_keystore. And below, the opt/ab2d directory gets created, but when does the file get moved?

Also, why is the ab2d_sbx_keystore in the bcda-opensbx-access-logs bucket?